### PR TITLE
Fearure/add gcc 11 container

### PR DESCRIPTION
**Pull request by KVS85**
Required to change default Linux CI compiler to gcc-11 for devel.

- [ ] ArangoDB PR: https://github.com/arangodb/arangodb/pull/16474/

### CODE REVIEW

File: containers/buildAlpine6.docker/tools/gcc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /usr/bin/sccache gcc "$@"
```
Comment (dothebart) [inline containers/buildAlpine6.docker/tools/gcc:2]: don't shellscripts need a final newline?

File: containers/buildUbuntu6.docker/tools/gcc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /usr/bin/sccache gcc "$@"
```
Comment (dothebart) [inline containers/buildUbuntu6.docker/tools/gcc:2]: \n?

**Review by dothebart — APPROVED** [review]
lgtm except for 2 missing newlines.

**Review by fceller — APPROVED** [review]

File: containers/buildAlpine6.docker/tools/gcc (hunk 1 above)
Comment (KVS85) [reply containers/buildAlpine6.docker/tools/gcc:2]: Should not harm. Worked fine with older containers.

File: containers/buildUbuntu6.docker/tools/gcc (hunk 2 above)
Comment (KVS85) [reply containers/buildUbuntu6.docker/tools/gcc:2]: Should not harm. Worked fine with older containers.

**Pull request merged**